src/main/java/io/confluent/examples/streams/microservices/AddInventory.java

```diff
@@ -46,19 +52,50 @@ private static void sendInventory(final List<KeyValue<Product, Integer>> invento
         }
     }
 
-    public static void main(final String[] args) {
+    public static void main(final String[] args) throws Exception {
+
+        final Options opts = new Options();
+        opts.addOption(Option.builder("b")
+                .longOpt("bootstrap-server").hasArg().desc("Kafka cluster bootstrap server string").build())
```

Comment:
@rspurgeon for consistency, consider making this plural `servers`

Reply:
done